libqtile/layout/xmonad.py

```diff
@@ -237,16 +237,16 @@ def cmd_normalize(self, redraw=True):
             self.group.layout_all()
         self.do_normalize = False
 
-    def cmd_reset(self, redraw=True):
+    def cmd_reset(self, ratio=None, redraw=True):
```

Comment:
Can you add a `cmd_set_ratio(ratio)` method which could call `self.cmd_reset(ratio=ratio)`.